src/Math-Matrix/PMMatrix.class.st

```diff
@@ -508,6 +508,7 @@ PMMatrix >> hash [
 { #category : #initialization }
 PMMatrix >> initialize: anInteger [
 	"Build empty components for a square matrix. No check is made: components are assumed to be orgainized in rows."
+	[ anInteger isInteger and: [ anInteger  >= 0 ] ] assert.
```

Comment:
Can we have a zero dimensional matrix? I don't think we do.

Reply:
I am not sure of this actually. But `Array2D` allows zero-dimensional matrix and we can create a `PMVector` of size 0. So, I felt we should allow zero-dimensional matrix in `PMMatrix`. What do you think?

Reply:
This is a domain question that may be worth raising with Serge or in discord

Reply:
Maybe the initialize method should be renamed after the discord conversation, and the comment for this method, which says clients get a square matrix